### PR TITLE
Slime Cake

### PR DESCRIPTION
**Pull request by ghost**
This is my first coding project, I added a slimecake. 

Like I said I am new to this so if there is anything that I left out please tell me.

### CODE REVIEW

**Comment by RemieRichards** [comment]
you named the PR your name, Cute.

**Comment by Cheridan** [comment]
:star:

**Comment by Ahammer18** [comment]
Yeah, I forgot to rename it

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,49 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"
```
Comment (RemieRichards) [inline tgstation.dme]: You need to untick "Automatically assign FILE_DIR"

**Comment by Ahammer18** [comment]
There we go

File: tgstation.dme (hunk 1 above)
Comment (Cheridan) [inline tgstation.dme]: Ahhhhh

**Comment by RemieRichards** [comment]
It doesn't need your name in it at all, haha this is so cute.

**Comment by Ahammer18** [comment]
:3

**Comment by Ahammer18** [comment]
Righto

**Comment by Ahammer18** [comment]
I'll re commit it, please take down this request

**Comment by RemieRichards** [comment]
@Ahammer18 just push to the same branch, the PR will update.

**Comment by Ahammer18** [comment]
There we go heuhuhuh

**Comment by RemieRichards** [comment]
:+1:

**Comment by Jordie0608** [comment]
You should make a changelog because I like changelogs and it's good to learn how.

**Comment by Ahammer18** [comment]
h-how

**Comment by RemieRichards** [comment]
@Jordie0608 leave the poor babby alone, suffered enough!

**Comment by RemieRichards** [comment]
Omg your branch is your name too somehow this just keeps getting cuter.

**Comment by ike709** [comment]
they grow up so fast, he's already made his first changelog

File: code/modules/food&drinks/food/snacks_cake.dm [hunk 2]
```diff
@@ -161,3 +161,16 @@
...
+	desc = "A cake made of slimes. Probably not electrified."
+	icon_state = "slimecake"
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/cakeslice/slimecake
+	slices_num = 5
+	list_reagents = list("nutriment" = 1, "vitamin" = 3)
+
+/obj/item/weapon/reagent_containers/food/snacks/cakeslice/slimecake
+	name = "Slime cake slice"
+	icon_state = "slimecake_slice"
+	filling_color = "#FA8072"
```
Comment (phil235) [inline code/modules/food&drinks/food/snacks_cake.dm]: I don't think this is the correct color. This is salmon, whereas your slime cake sprite color is pretty close to cyan.

File: html/changelogs/Ahammer18-PR-8215.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+author: Ahammer18
+delete-after: true
+
+changes: 
+  - rscadd: "Adds the braincake and braincake slice."
```
Comment (phil235) [inline html/changelogs/Ahammer18-PR-8215.yml]: Don't you mean Slimecake?

**Comment by Mandurrrh** [comment]
Woooo hoo welcome to the club! :balloon:

**Comment by Ahammer18** [comment]
I'll change that stuff when I get home, the color is left over from when I looked at the other cakes for an example, my change log is messed up because it was really late and I was tired.

**Pull request merged**